scenarios/bigquery_configgen.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Very simple script to generate JSON metric config files. Exists to handle escaping sql and jq."""
+
+import argparse
+import json
+import bigquery
+
+def main(query_file, jq_file, metric, out_file):
+    """Generates a config file defining a metric."""
+    bigquery.validate_metric_name(metric)
+    query = query_file.read()
+    query_file.close()
```

Comment:
no need to close the file explicitly-- it will happen when the program exits, which is almost instantaneous